specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-02-10/blob.json

```diff
@@ -6125,9 +6125,6 @@
           },
           {
             "$ref": "#/parameters/BlobTagsHeader"
-          },
-          {
-            "$ref": "#/parameters/SealBlob"
```

Comment:
do we also want to remove SealBlob? https://github.com/Azure/azure-rest-api-specs/blob/9ef4535351b826bccf3deb6d35964ad10e7e3c69/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-02-10/blob.json#L11789

Reply:
no, seal blob is used in some other APIs still

Reply:
you are right!!!